test/sql/validation.sql

```diff
@@ -187,6 +187,8 @@ SELECT COUNT(*) FROM test_validation GROUP BY LENGTH(city);
 SELECT COUNT(*) FROM test_validation GROUP BY city || 'xxx';
 SELECT LENGTH(city) FROM test_validation;
 SELECT city, 'aaaa' FROM test_validation GROUP BY 1, 2;
+PREPARE prepared_param_as_label(text) AS SELECT city, $1 FROM test_validation GROUP BY 1, 2;
+EXECUTE prepared_param_as_label('aaaa');
```

Comment:
I would add some tests with actual data to see if anonymization indeed happens with prepared statements.

Reply:
Yup, that was understood. I'll add such a test